velox/exec/tests/PartitionedOutputBufferManagerTest.cpp

```diff
@@ -420,3 +421,21 @@ TEST_F(PartitionedOutputBufferManagerTest, serializedPage) {
     EXPECT_EQ(mappedMemory->allocateBytesStats().totalSmall, 0);
   }
 }
+
+TEST_F(PartitionedOutputBufferManagerTest, getDataOnTaskWithNoBuffer) {
```

Comment:
nit: perhaps, rename to getDataOnFailedTask and add comments to explain the scenario this test is verifying.